generator/sbpg/targets/rust.py

```diff
@@ -23,6 +23,33 @@
 MESSAGES_TEMPLATE_NAME = "sbp_messages_template.rs"
 MESSAGES_MOD_TEMPLATE_NAME = "sbp_messages_mod.rs"
 
+GPS_TIME = """\
```

Comment:
Mentioned offline, but if the type uses GPSTime, then we potentially have the "nanosecond residual" available... but digging in to this deeper I don't think we need to worry about this (maybe?).  The bare `GPSTime` object is only used in `MSG_TRACKING_STATE_DETAILED_DEP_A`, `MSG_SPECAN`, and (the only one that might matter) is `MSG_OBS`.

Reply:
It's not really a technical problem since the two structs with the same name are defined in different crates, so they're in different name spaces. It's mostly a potential cause for confusion among developers, right? I'd like to think that's less likely to happen since the libsbp `GPSTime` struct typically won't be instantiated directly. I imagine it would be accessed as a member of a larger SBP message most of the time. Maybe some clarification in the docs would help?

Reply:
> It's not really a technical problem since the two structs with the same name are defined in different crates, so they're in different name spaces. It's mostly a potential cause for confusion among developers, right? I'd like to think that's less likely to happen since the libsbp `GPSTime` struct typically won't be instantiated directly. I imagine it would be accessed as a member of a larger SBP message most of the time. Maybe some clarification in the docs would help?

Yeah, that's probably sufficient, I was also thinking a wrapper type might be appropriate to disambiguate `MessageTime(GpsTime)` or something similar.